### PR TITLE
Update openscap docs - tailoring files

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

File: plugins/foreman_openscap/0.6/index.md [hunk 1]
```diff
@@ -23,17 +23,21 @@ Foreman OpenSCAP plugin is made of 5 components:
 
 ## 1.1 OpenSCAP basic concepts
 
-There are three basic concepts (entities) in OpenSCAP plug-in: SCAP Contents, Compliance Policies and ARF Reports.
+There are three basic concepts (entities) in OpenSCAP plug-in: SCAP Contents, Compliance Policies, ARF Reports and Tailoring Files.
```
Comment (ares) [inline plugins/foreman_openscap/0.6/index.md]: s/three/four/

**Review by ares — CHANGES_REQUESTED** [review]
except one nit looks good, grammar check by other reviewer would be appreciated

**Comment by xprazak2** [comment]
I updated the number of components

**Review by ares — APPROVED** [review]

**Comment by mmoll** [comment]
merged, díky @xprazak2!

**Pull request merged**